docs/geofabrics/update-geofabric.md

```diff
@@ -3,9 +3,33 @@ sidebar_position: 20
 title: Update GeoFabrics
 ---
 
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+This page lists several methods for updating GeoFabrics. Which GeoFabrics you can access depends on your assigned [Permissions](../account-management/permissions/index.md).
+
+<Tabs groupId="operating-systems">
+<TabItem value="console" label="Web Console">
+
 To modify a geofabric:
 
 1. Log in to an administrator account in the `_system` GeoFabric.
 2. Click **Geo Fabrics** to navigate to the GeoFabrics management page.
 3. Click the GeoFabric that you want to modify.
 4. Add regions to or remove regions from the GeoFabric by clicking the **Status** switch next to each region.
+
+</TabItem>
+<TabItem value="api" label="REST API">
+
+Use our interactive API Reference with code generation in 18 programming languages to:
```

Comment:
```suggestion
Use our interactive API reference with code generation in 18 programming languages to:
```

Are we capitalizing the R in "API reference"?

Reply:
Since it is the actual title of the page, yes.